### PR TITLE
37596 PDF Download in property lists

### PR DESCRIPTION
**Pull request by yeneastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/590

changed log :
Remove PDF download in property lists and unit lists.

### CODE REVIEW

**Review by andernath — COMMENTED** [review]
@yeneastgate @dai-eastgate 
I finished review, looks good. 
Please check my comment.

File: plugin/Gui/AdminPageEstateListSettings.php [hunk 1]
```diff
@@ -169,14 +169,6 @@ protected function buildForms()
 		$pFormModelPictureTypes->addInputModel($pInputModelPictureTypes);
 		$this->addFormModel($pFormModelPictureTypes);
 
-		$pInputModelDocumentTypes = $pFormModelBuilder->createInputModelExpose();
-		$pFormModelDocumentTypes = new FormModel();
-		$pFormModelDocumentTypes->setPageSlug($this->getPageSlug());
-		$pFormModelDocumentTypes->setGroupSlug(self::FORM_VIEW_DOCUMENT_TYPES);
```
[the remark this answers is not in the document]
Comment (yeneastgate) [reply plugin/Gui/AdminPageEstateListSettings.php:175]: @andernath I will check and remove it.

**Review by andernath — APPROVED** [review]
I will start testing

**Comment by fredericalpers** [comment]
Nach erneuter Rücksprache soll das Feature hier weiterhin entfernt werden, da es an dieser Stelle keinen Sinn macht.

**Review by andernath — APPROVED** [review]

**Pull request merged**